.pre-commit-hooks.yaml

```diff
@@ -1,7 +1,7 @@
--   id: fix-links
+-   id: fix-capella-fragment-links
```

Comment:
As this repository is for capella-git-hooks, I don't think it is necessary to include capella in the hook ID. I get the point of adding `fragment` to the ID, but I am not sure whether this justifies the additional work it takes to change the hook in the pre-commit config of the projects...

Reply:
Looking at this scope, yes, but the hook ID is also used in the (pre-commit wide) `SKIP` environment variable. That's why I included "capella" there. See the commit command that this script executes (translated to bash):

```
SKIP=fix-capella-fragment-links git commit -m "Fix links yada yada"
```